tests/benchmarks/popl18/lib/prover-ple-lib.cabal

```diff
@@ -0,0 +1,25 @@
+cabal-version:      2.4
```

Comment:
This is just a random little library, since I needed to use these two files in tests that were `TFSafe` and `TFUnsafe`, so they couldn't be part of the `TFUnsafe` ones (since then they would be expected to not compile).